### PR TITLE
new geo ip service

### LINKED ISSUE #288: We need a better geoip solution
How about we switch to https://ipgeolocation.io/ so all our GCP nodes don't show up in [Bikini Atoll](https://en.wikipedia.org/wiki/Bikini_Atoll):
image

### PR DESCRIPTION
**Pull request by manuel-calavera**
@mvines suggested to use different geo ip solution.  It should fix geo coord for edge validators. Fixes #288 

@sunnygleason @mvines please take a look. and i will reset my api key as soon as we merge it (if we merge it).

### CODE REVIEW

File: api/api.js [hunk 1]
```diff
@@ -31,12 +32,15 @@ const CLUSTER_INFO_BROADCAST_INTERVAL_MS = 5000;
 const CLUSTER_INFO_CACHE_TIME_SECS = 4500;
 const CONFIG_PROGRAM_ID = 'Config1111111111111111111111111111111111111';
 const MAX_KEYBASE_USER_LOOKUP = 50;
+const GEO_IP_KEY = '2504e6097a8048c585560d1ebeff00c2';
```
Comment (manuel-calavera) [inline api/api.js]: i will reset it, pls create your own and set it as required.

Comment (mvines) [reply api/api.js]: Can we do something like:
https://github.com/solana-labs/tour-de-sol/blob/1b2e1f041a0a52d5f28d01c8071a9e3fb49a0480/ip-collector/audit.js#L7-L13
except that when there's no key just emit a warning and disable geolocation

**Comment by sunnygleason** [comment]
@manuel-calavera this seems reasonable -- if you have time to implement @mvines suggestion about "if no geoip key, emit warning on startup, return null for lookups", that would be excellent!

**Comment by manuel-calavera** [comment]
@sunnygleason please check if it looks good.

File: api/api.js (hunk 1 above)
Comment (manuel-calavera) [reply api/api.js]: @mvines done

**Comment by sunnygleason** [comment]
@manuel-calavera thank you, just one (hopefully) last question!

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "http-proxy": "^1.18.0",
     "humps": "^2.0.1",
     "ip": "^1.1.5",
+    "ip-geolocation-api-javascript-sdk": "^1.0.7",
```
Comment (mvines) [inline package.json]: The `geoip-lite` dependency is now unused right?  Let's remove it

Comment (manuel-calavera) [reply package.json]: thanks, done

**Pull request merged**